tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -13191,6 +13196,8 @@ bool SpirvEmitter::emitEntryFunctionWrapper(const FunctionDecl *decl,
   auto *entryLabel = spvBuilder.createBasicBlock();
   spvBuilder.setInsertPoint(entryLabel);
 
+  processInlineSpirvAttributes(decl);
+
```

Comment:
Why did you move the call to this function? I'm fine with it here, but if it needs to be called after something else happens, then we should document it with a comment.

Reply:
Sure, I added a comment:

> Handle vk::execution_mode, vk::ext_extension and vk::ext_capability attributes. Uses pseudo-instructions for extensions and capabilities, which are added to the beginning of the entry basic block, so must be called after the basic block is created and insert point is set.